src/app/client/src/app/modules/core/components/content-type/content-type.component.ts

```diff
@@ -113,8 +114,7 @@ export class ContentTypeComponent implements OnInit, OnDestroy {
 
   processFormData(formData) {
     this.contentTypes = _.sortBy(formData, 'index');
-    const defaultTab = _.find(this.contentTypes, ['default', true]);
-    this.selectedContentType = this.activatedRoute.snapshot.queryParams.selectedTab || _.get(defaultTab, 'contentType') || 'textbook';
+    this.selectedContentType = this.activatedRoute.snapshot.queryParams.selectedTab || 'textbook';
```

Comment:
Not sure why piece of code is removed. It's used to select the default tab based on form config.

const defaultTab = _.find(this.contentTypes, ['default', true]);